staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -866,54 +882,3 @@ func NewDefaultAPIGroupInfo(group string, scheme *runtime.Scheme, parameterCodec
 		NegotiatedSerializer:   codecs,
 	}
 }
-
-// getOpenAPIModels is a private method for getting the OpenAPI models
-func (s *GenericAPIServer) getOpenAPIModels(apiPrefix string, apiGroupInfos ...*APIGroupInfo) (openapiproto.Models, error) {
-	if s.openAPIConfig == nil {
-		return nil, nil
-	}
-	pathsToIgnore := openapiutil.NewTrie(s.openAPIConfig.IgnorePrefixes)
-	resourceNames := make([]string, 0)
-	for _, apiGroupInfo := range apiGroupInfos {
-		groupResources, err := getResourceNamesForGroup(apiPrefix, apiGroupInfo, pathsToIgnore)
-		if err != nil {
-			return nil, err
-		}
-		resourceNames = append(resourceNames, groupResources...)
-	}
-
-	// Build the openapi definitions for those resources and convert it to proto models
-	openAPISpec, err := openapibuilder2.BuildOpenAPIDefinitionsForResources(s.openAPIConfig, resourceNames...)
-	if err != nil {
-		return nil, err
-	}
-	for _, apiGroupInfo := range apiGroupInfos {
-		apiGroupInfo.StaticOpenAPISpec = openAPISpec
```

Comment:
This is the old `StaticAPISpec` that was used in `buildProtoModelsForApply`. Just pointing out that It was formerly constructed with `BuildOpenAPIDefinitionsForResources` and a whitelist of required names.

The provided openapiConfig's `GetDefinitions` may be the `generatedopenapi` but it also could have been customized. TODO: This customizing functionality is currently being lost.

The new `StaticAPISpec` uses `generatedopenapi.GetDefinitions` to construct it directly from the apiextefnsion-apiserver's definitions. This required also changing update-codegen to include resource.Quantity. 

If we could use `s.StaticOpenAPISpec` instead, then that would also respect the customization. However, `s.StaticOpenAPISpec` is not available until `PrepareRun`, which occurs after `InstallAPIs`.

One thing to note is that the spec used by the custom resource handler is not read from until an HTTP serving call. It may be possible to set the value at a later time.

@apelisse advice?